chart/templates/frontend_deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "gmp.name" . }}-{{ .Values.frontend.name }}
```

Comment:
I'd suggest we put this into the gmp-public namespace as its a deployment meant to be talked to by other services in the cluster and not wihtin the set managed by the operator.

Reply:
Indeed, we can move the frontend component from `gmp-system` namespace to `gmp-public` it's relevant.